packages/itinerary-body/src/styled.tsx

```diff
@@ -302,13 +302,17 @@ export const LegDescriptionHeadsignPrefix = styled.span`
  * Lets others apply styles to the mode text in
  * "Bicycle 0.5 miles to City Hall"
  */
-export const LegDescriptionMode = styled.span``;
+export const LegDescriptionMode = styled.span`
+  font-weight: inherit;
```

Comment:
Unrelated change, please revert and move to another PR (2 instances).

Reply:
This is needed for the snapshots to pass